pom.xml

```diff
@@ -418,22 +422,7 @@
       <dependency>
         <groupId>org.apache.parquet</groupId>
         <artifactId>parquet-avro</artifactId>
-        <!--
-            IMPORTANT NOTE
-            ==============
-            Spark 2.2.0 bumps to Parquet 1.8.2, but keeps Avro at 1.7.3. This
-            causes an issue when using parquet-avro 1.8.2, which relies on a
-            method that is new in Avro 1.8.x. The recommended workaround is to
-            pin parquet-avro to 1.8.1, instead of 1.8.2, as described in the
-            Spark 2.2.0 release notes.
-        -->
-        <version>1.8.1</version>
```

Comment:
This is exciting!

Reply:
There still may be cases where runtime exceptions are thrown because of conflicting classpath problems, if so the shade workaround might also be necessary.